patterns/contact-info-locations.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Title: Contact: Info and locations
+ * Slug: twentytwentyfive/contact-info-locations
+ * Keywords: contact, location
+ * Categories: contact
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|50","left":"var:preset|spacing|50","top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
+	<!-- wp:heading {"textAlign":"left","align":"full","fontSize":"xx-large"} -->
+	<h2 class="wp-block-heading alignfull has-text-align-left has-xx-large-font-size">How to get in touch with us</h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"},"blockGap":"0","margin":{"top":"var:preset|spacing|50"}},"border":{"top":{"color":"var:preset|color|primary","width":"1px"},"right":{},"bottom":{},"left":{}}},"layout":{"type":"grid","minimumColumnWidth":"23rem"}} -->
+	<div class="wp-block-group" style="border-top-color:var(--wp--preset--color--primary);border-top-width:1px;margin-top:var(--wp--preset--spacing--50);padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)">
+		<!-- wp:group {"style":{"layout":{"columnSpan":1,"rowSpan":1}},"layout":{"type":"flex","orientation":"vertical"}} -->
+		<div class="wp-block-group">
+			<!-- wp:heading {"level":3,"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"small"} -->
+			<h3 class="wp-block-heading has-small-font-size" style="font-style:normal;font-weight:700">Social media</h3>
+			<!-- /wp:heading -->
+			<!-- wp:navigation {"overlayMenu":"never","style":{"spacing":{"blockGap":"0"}},"fontSize":"small","layout":{"type":"flex","orientation":"vertical"},"ariaLabel":"<?php esc_attr_e( 'Social', 'twentytwentyfive' ); ?>"} -->
```

Comment:
The line height appears to be different in Figma. We should probably tackle that by setting it at a block level.

Reply:
I tried removing the block spacing and setting a low line height, but it broke the outline:
![image](https://github.com/user-attachments/assets/f1c32741-ebf0-4a07-a427-fe5157eab14f)

Reply:
Ok, we can revisit this one later. Thank you.